src/plugins/chatwizard/chatwizard.js

```diff
@@ -475,17 +529,28 @@ var componentName = "wb-chtwzrd",
 	 * @param {String} title The title of the wizard window, as well as the notification
 	 */
 	buildChtwzrd = function( $selector, title ) {
-		$selector.after( "<div class='" + componentName + "-bubble-wrap'><a href='#" + componentName + "-container aria-controls='" + componentName + "-container class='" + componentName + "-link bubble trans-pulse' role='button'>" + i18nDict.trigger + "</a>" + ( !isNotif ? "<p class='trans-left'><span class='notif'>" + title +
-		"</span> <a href='#' class='notif-close' title='" + i18nDict.notification + "' aria-label='" + i18nDict.notification + "' role='button'>&times;</a></p>" : "" ) + "</div>" );
+		$selector.after( "<div class='" + componentName + "-bubble-wrap'><a href='#" + componentName + "-container' aria-controls='" + componentName + "-container' class='" + componentName + "-link bubble trans-pulse' role='button'>" + i18nDict.trigger + "</a>" + ( !isNotif ? "<p class='trans-left'><span class='notif'>" + title + "</span> <a href='#' class='notif-close' title='" + i18nDict.notification + "' aria-label='" + i18nDict.notification + "' role='button'>Ã—</a></p>" : "" ) + "</div>" );
 		$selector.next( selector + "-bubble-wrap" ).after( "<aside id='" + componentName + "-container' class='modal-content overlay-def " + componentName + "-container " + ( isInline ? " wb-chtwzrd-contained" : "" ) + "'></aside>" );
 
 		var $container = $( selector + "-container" );
-		$container.append( "<header class='modal-header header'><h2 class='modal-title title'>" + title + "</h2><button type='button' class='minimize' title='" + i18nDict.minimize + "'><span class='glyphicon glyphicon-chevron-down'></span></button></header>" );
+		$container.append(
+			"<header class='modal-header header'><h2 class='modal-title title'>" +
+			title + "</h2><button type='button' class='reset btn-reset' title='" +
```

Comment:
You do not need to have reset and btn-reset, just stick with reset:
```suggestion
			title + "</h2><button type='button' class='reset' title='" +
```